src/server.ts

```diff
@@ -206,7 +206,7 @@ const getCAmsg = (deps, diagnostics): string => {
 };
 
 const caDefaultMsg = 'Checking for security vulnerabilities ...';
-
+const custom_header = JSON.parse(process.env.CUSTOM_HEADER);

```

Comment:
Follow the existing coding standard of the file.

Use camelcase for naming variables.